libs/ui-lib/lib/common/components/clusterWizard/ClusterWizardStepValidationsAlert.tsx

```diff
@@ -111,6 +116,16 @@ const ClusterWizardStepValidationsAlert = <ClusterWizardStepsType extends string
           </Alert>
         </AlertGroup>
       )}
+      {lastInstallationPreparation && lastInstallationPreparation.status === 'failed' && (
+        <AlertGroup>
+          {children}
```

Comment:
Is the rendering of the `children` here correct?


They are included here: https://github.com/openshift-assisted/assisted-installer-ui/blob/07b0eda38e3079544916d69c7ad10e0713d2d545/libs/ui-lib/lib/common/components/clusterWizard/ClusterWizardStepValidationsAlert.tsx#L85

This is only used in CIM, and since it's passing `ValidationsRunningAlert`, I think it wouldn't make sense to have the two alerts at the same time.

Reply:
I've removed {children}